package.json

```diff
@@ -32,13 +33,13 @@
     "@ambassify/backoff-strategies": "1.0.0",
     "bat-balance": "^1.0.7",
     "bat-publisher": "^2.0.9",
-    "bitgo": "^4.15.0",
     "brave-crypto": "^0.1.0",
     "http-request-signature": "0.0.2",
     "joi": "^13.1.2",
     "json-stable-stringify": "1.0.1",
     "node-anonize2-relic-emscripten": "^0.3.3",
     "random-lib": "2.1.0",
+    "snazzy": "^7.1.1",
```

Comment:
please move this dependency to `devDependencies`

Reply:
Done